Gems/Atom/Feature/Common/Code/Source/Debug/RayTracingDebugFeatureProcessor.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/RPI.Public/RenderPipeline.h>
+#include <Debug/RayTracingDebugFeatureProcessor.h>
+#include <RayTracing/RayTracingFeatureProcessor.h>
+
+namespace AZ::Render
+{
+    void RayTracingDebugFeatureProcessor::Reflect(ReflectContext* context)
+    {
+        if (auto* serializeContext{ azrtti_cast<SerializeContext*>(context) })
+        {
+            // clang-format off
+            serializeContext->Class<RayTracingDebugFeatureProcessor, RPI::FeatureProcessor>()
+                ->Version(0)
+            ;
+            // clang-format on
```

Comment:
Any specific reason, why you don't let clang format this?

Reply:
For style and indentation consistency, because in most cases `->Version()` is in a separate line even if it would fit on the same line